README.md

```diff
@@ -1,23 +1,39 @@
 # CS Club Website
 
-## Development
+---
 
-1. Install dependencies
+This is the official repository for the University of Adelaide Computer Science Club (CS Club) website. It is built using [Next.js](https://nextjs.org/), [React](https://reactjs.org/), [TypeScript](https://www.typescriptlang.org/), and [Tailwind CSS](https://tailwindcss.com/).
+
+## Getting Started
+
+To get started, please follow these steps: 
+1. Clone the repository and open the folder.
```

Comment:
Remove this step. I assume it's dumplicated with contributing guide. Also, it's recommanded to use ssh to clone a git repo.